### DISCUSSION TITLE
[Docker] Allow `DEFAULT_GROUP` to be private on setup

### DISCUSSION THREAD

**Discussion by ToshY** [mealie-recipes/mealie#3241] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Currently the created default group "Home" creates a public group on setup with docker, meaning that every non-authenticated user can view it.

### Please provide a concise description of the feature that would resolve your issue.

I'd like see one of the following:

- Make the default group visibility private by default.
- Have another environment variable to set default group visibility. 

### Please consider and list out some caveats or tradeoffs made in your design decision

n/a

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by hay-kot** [comment]
What's wrong with having to change this after you login?

**Reply by ToshY** [reply]
Nothing "wrong" with that. This would just make it easier (and less error prone for people/me) if it can be automated by having the ability to set this by default.
